src/main/java/uk/gov/ons/ctp/response/collection/exercise/client/impl/ActionSvcRestClientImpl.java

```diff
@@ -160,6 +161,56 @@ public ActionPlanDTO getActionPlanBySelectors(
     return actionPlans.get(0);
   }
 
+  @Retryable(
+      value = {RestClientException.class},
+      maxAttemptsExpression = "#{${retries.maxAttempts}}",
+      backoff = @Backoff(delayExpression = "#{${retries.backoff}}"))
+  @Override
+  public List<ActionPlanDTO> getActionPlansBySelectorsSocial(final String collectionExerciseId) {
+    log.debug(
+        "Retrieving action plan for selectors, collectionExerciseId: {}", collectionExerciseId);
+
+    final MultiValueMap<String, String> queryParams = new LinkedMultiValueMap<>();
+    queryParams.add(SELECTOR_COLLECTION_EXERCISE_ID, collectionExerciseId);
+    final UriComponents uriComponents =
+        restUtility.createUriComponents(appConfig.getActionSvc().getActionPlansPath(), queryParams);
+
+    HttpEntity httpEntity = restUtility.createHttpEntity(null);
+
+    final ResponseEntity<List<ActionPlanDTO>> responseEntity;
+    responseEntity =
+        restTemplate.exchange(
+            uriComponents.toString(),
+            HttpMethod.GET,
+            httpEntity,
+            new ParameterizedTypeReference<List<ActionPlanDTO>>() {});
+
+    log.debug(
```

Comment:
I see a lack of info in this class , there is any chance to log at info level maybe this message ?